### PR TITLE
Update script relate data to CLDR 36 and UCD 13.0.0

### PR DESCRIPTION
**Pull request by gouniLee**
### Checklist

* [ ] At least one test case is included for this feature or bug fix.
* [ ] `ReleaseNotes` has added or is not needed.
* [x] This PR has passed all of the test cases on `Nodejs` and `Chrome Browser`.
* [x] This PR has passed all of the test cases on `QT/QML`.
* [ ] This is an API breaking change.
* [ ] Requires a major version change.

### Issue Resolved / Feature Added
`scriptToRange.json` `rangeToScript.json`, and `scripts.jf` files are updated to CLDR 36, Unicode 13.0.0  by running `genscritps.js`, `genlangscripts.js`.
`scripts.json` hasn't changed since the last updated version.

### Links
https://unicode.org/iso15924/iso15924-text.html
http://unicode.org/Public/cldr/36/ --> common/properties/scriptMetadata.txt

### CODE REVIEW

File: js/data/locale/bh/scripts.jf [hunk 1]
```diff
@@ -1,7 +0,0 @@
-{
```
Comment (ehoogerbeets) [inline js/data/locale/bh/scripts.jf]: I think Behari is still written in Devanagari, right? Why are these files deleted?

**Comment by gouniLee** [comment]
Regarding deleted files, I think those were not in CLDR (`cldr-data/supplementa/languageData.json`) and It wasn't properly deleted last cldr34 update. It should've been deleted last time work. The hard part is that CLDR data isn't easy to figure out the change if it's not a major change.

**Comment by gouniLee** [comment]
FYI.
Deleted languages are associated with data in [supplementtalMetadata.xml](https://github.com/unicode-org/cldr/blob/master/common/supplemental/supplementalMetadata.xml) 
i.e)
```
<languageAlias type="kpv" replacement="kv" reason="macrolanguage"/> <!-- Komi-Zyrian ⇒ Komi -->
```
The data associated with them use the replacement value. So kpv data is found under ‘kv’.
